portfolio/src/main/java/com/google/sps/servlets/LikeDataServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+/** Servlet responsible for liking comments. */
+@WebServlet("/like-data")
+public class LikeDataServlet extends HttpServlet {
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long likes = Long.parseLong(request.getParameter("likes"));
+    long id = Long.parseLong(request.getParameter("id"));
+    likes ++;
```

Comment:
Curious: why increment?

Reply:
Because with each call of POST the number of likes increases by 1. Would it make more sense if the incremented number was already passed as the parameter?

Reply:
Okay, I did decide to change it so the incremented number of likes was already passed as a parameter for POST.